docs/3___packaging.adoc

```diff
@@ -23,6 +23,8 @@ This allows the bundling of multiple variants of a system structure definition r
 There *MUST* still be one SSD file named SystemStructure.ssd at the root of the ZIP archive which is deemed to be the default variant.
 Tools not supporting variant-containing SSPs *MUST* deal with the given SSP by ignoring all contained SSD files except for the SystemStructure.ssd default file.
 
+If the ZIP archive contains multiple SSD file entries at the root level, the `name` attribute of each such system structure definition *MUST* be different from any of the other root level system structure definitions.
+
```

Comment:
Maybe warn tools that this requirement is easily violated, and has been newly added:

```suggestion

{empty} [ _Tools *MAY* want to be prepared to handle SSP archives that violate the unique name requirement, as this was not present in earlier releases of SSP, and manually generated SSP archives may inadvertantly violate this requirement._ ]

```